### PR TITLE
Add elasticsearch version as a prefix for the staging URL

### PR DESCRIPTION
**Pull request by s1monw**
This is purely for maintainance reasons since it easier to see if we can drop
certain stageing urls if we have the version next to the hash.
I also removed the gpg passphrase from the example URL since it's better to get prompted?

@spinscale can you take a look?

### CODE REVIEW

**Comment by dadoonet** [comment]
It makes sense and looks good to me.

**Comment by rjernst** [comment]
LGTM

File: dev-tools/prepare_release_create_release_version.py [hunk 1]
```diff
@@ -142,4 +142,4 @@ def find_release_version():
   shortHash = subprocess.check_output('git log --pretty=format:"%h" -n 1', shell=True)
   print('')
   print('*** To create a release candidate run: ')
-  print('  mvn clean deploy -Prelease -DskipTests -Dgpg.keyname="$GPG_KEY_ID" -Dgpg.passphrase="$GPG_PASSPHRASE" -Dpackaging.rpm.rpmbuild=/usr/bin/rpmbuild -Delasticsearch.s3.repository=s3://download.elasticsearch.org/elasticsearch/staging/%s' % (shortHash.decode('utf-8')))
```
Comment (spinscale) [inline dev-tools/prepare_release_create_release_version.py]: why did you remove the `GPG_PASSPHRASE` here?

**Comment by spinscale** [comment]
left one comment, LGTM

File: dev-tools/prepare_release_create_release_version.py (hunk 1 above)
Comment (s1monw) [inline dev-tools/prepare_release_create_release_version.py]: the commit msg says:

> I also removed the gpg passphrase from the example URL since it's better to get prompted?

Comment (spinscale) [inline dev-tools/prepare_release_create_release_version.py]: sorry for that. saturday morning reviews are not my thing. Due to the potential problem of having to enter the passphrase more than once, I'd suggest we print out two different mvn lines and let the user choose...

File: dev-tools/prepare_release_create_release_version.py [hunk 2]
```diff
@@ -142,4 +142,6 @@ def find_release_version():
...
+  print('  mvn clean deploy -Prelease -DskipTests -Dgpg.keyname="D88E42B4" -Dpackaging.rpm.rpmbuild=/usr/bin/rpmbuild -Delasticsearch.s3.repository=s3://download.elasticsearch.org/elasticsearch/staging/elasticsearch-%s-%s' % (release_version, shortHash.decode('utf-8')))
+  print('');
+  print('NOTE: this command will promt you several times for the GPG passphrase of the key you specified you can alternatively pass it via -D-Dgpg.passphrase=yourPassPhrase');
```
Comment (spinscale) [inline dev-tools/prepare_release_create_release_version.py]: `-D-D` ?

**Comment by spinscale** [comment]
one last nitpicking comment, LGTM, push it!

File: dev-tools/prepare_release_create_release_version.py (hunk 2 above)
Comment (s1monw) [inline dev-tools/prepare_release_create_release_version.py]: hmm odd - I will fix
